ios/RNOneginiSdk.swift

```diff
@@ -43,18 +43,49 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
     override static func requiresMainQueueSetup() -> Bool {
         return true
     }
-
-    @objc
-    func startClient(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
-        self.oneginiSDKStartup { _, error in
+    
+    func rejectWithError(_ reject: RCTPromiseRejectBlock, _ error: Error) {
```

Comment:
99% if you are using `rejectWithError` only with `WrapperError` you can define it here and use only `.authenticatorDoesNotExist` in calls.

Reply:
We are using this with other Errors aswell.